Broken due to: [ERROR] Runtime.ImportModuleError: Unable to import module 'index': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with OpenSSL 1.0.2k-fips  26 Jan 2017.

Currently getting this error in Cloudwatch Logs with a GET to /serverless_lambda_stage. 

> [ERROR] Runtime.ImportModuleError: Unable to import module 'index': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with OpenSSL 1.0.2k-fips  26 Jan 2017. See: https://github.com/urllib3/urllib3/issues/2168

--- a/zip_based_lambda_functions/api-lambda-dynamodb-example/src/requirements.txt
+++ b/zip_based_lambda_functions/api-lambda-dynamodb-example/src/requirements.txt
@@ -1,2 +1,3 @@
 requests
 ptvsd
+urllib3==1.26.15

